docs/usage/rtk-query/queries.md

```diff
@@ -177,7 +177,7 @@ What you'll see below is this:
 Click the 'Add bulbasaur' button. You'll observe the same behavior described above until you click the 'Refetch' button on one of the components.
 :::
 
-<iframe src="https://codesandbox.io/embed/concepts-queries-deduping-caching-5qy3n?fontsize=12&hidenavigation=1&theme=dark"
+<iframe src="https://codesandbox.io/embed/github/reduxjs/redux-toolkit/tree/examples/rtk-query/src/query/examples/concepts-queries-deduping-caching?fontsize=12&hidenavigation=1&theme=dark"
```

Comment:
This would get updated to something like `https://codesandbox.io/embed/github/reduxjs/redux-toolkit/tree/master/src/query/examples/concepts-queries-deduping-caching` once merged.